XMonad/Hooks/StatusBar.hs

```diff
@@ -339,6 +339,9 @@ statusBarPipe cmd xpp  = do
 -- property, their content will be the same. If you want to use property-based
 -- logging with multiple bars, they should read from different properties.
 --
+-- "XMonad.Util.Loggers" includes loggers that can be bound to specific screens,
+-- like 'logCurrentOnScreen', that might be useful with multiple screens.
+--
```

Comment:
Actually, depending how #535 goes we might mention othes loggers here as well

Reply:
Alright, let's keep this open till we sort that out

Reply:
Now that that's done it may be nice to mention `logTitles` here

Reply:
I mentioned this in `X.H.StatusBar.PP` instead, it seemed more convenient